src/test/java/cz/cvut/kbss/study/persistence/dao/PatientRecordDaoTest.java

```diff
@@ -57,12 +55,24 @@ public class PatientRecordDaoTest extends BaseDaoTestRunner {
     @Autowired
     private InstitutionDao institutionDao;
 
+    @Autowired
+    private RoleGroupDao roleGroupDao;
+
+    RoleGroup roleGroupAdmin;
+
+    @BeforeEach
+    public void setUp() {
+        this.roleGroupAdmin = Generator.generateRoleGroupWithRoles(Role.administrator);
+        transactional(() -> roleGroupDao.persist(roleGroupAdmin));
+        int a =4;
```

Comment:
?

Reply:
Sorry, I forgot to remove it when I was debugging. Will be removed

Reply:
done